changes/cmd/changetool/editor.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+func editTemplate(template string) (string, error) {
+	editor := os.Getenv("VISUAL")
+	if editor == "" {
+		editor = "vim"
+	}
+
+	f, err := ioutil.TempFile(".", "tmp-template-entry")
+	if err != nil {
+		return "", err
+	}
+	defer os.Remove(f.Name())
+
+	_, err = f.Write([]byte(template))
```

Comment:
In general, should we pass around the template as just `[]byte`, given that most of the usage for marshaling / unmarshaling is using a byte slice rather then a string? This also reduces excess allocations.